developer-docs/plugin-guide.md

```diff
@@ -236,42 +184,29 @@ from pyglui import ui
 
 
 class Custom_Plugin(Plugin):
+    icon_chr = '@'  # custom menu icon symbol
```

Comment:
maybe comment about pupil-icon-font is better suited to be placed here as a comment?